zamba/object_detection/yolox/assets/yolox_tiny_640_20220528_model_kwargs.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
These two files need [to go in the MANIFEST.in](https://github.com/drivendataorg/zamba/blob/master/MANIFEST.in#L2) otherwise they won't be in the distributed version of the package.